web/themes/custom/hoeringsportal/templates/ds/ds-2col-stacked--node-citizen-proposal.html.twig

```diff
@@ -36,6 +36,49 @@
     <div class="row">
       <{{ left_wrapper }}{{ left_attributes.addClass('content group-left col-md-8') }}>
         <div class="content__main ">
+
+          {# Status message rejected #}
+          {% if node.field_content_state.value == 'rejected' %}
+            <div class="alert alert-warning border-0 d-flex gap-3" role="alert">
+              <i class="fas fa-triangle-exclamation mt-1"></i>
+              <div>
+                <h4 class="alert-heading mb-1 mt-0">{{ 'Rejected by the council'|t }}</h4>
+                <p class="mb-0">{{ 'The proposal was rejected at the citycouncil meeting @meeting_date'|t({'@meeting_date': node.field_city_council_meeting_date.value}) }}</p>
```

Comment:
We need to format the date along the lines of https://github.com/itk-dev/hoeringsportal/blob/develop/web/themes/custom/hoeringsportal/templates/content/node--teaser.html.twig#L112C47-L112C92.